integrated-benchmarks/express-otel/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "@opentelemetry/integrated-benchmarks-express-otel",
+  "version": "1.25.0",
+  "description": "OpenTelemetry Integrated Express Server Benchmarks",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js",
+  "scripts": {
+    "start": "node ./build/src/index.js",
+    "compile": "tsc --build",
+    "clean": "tsc --build --clean",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "version": "node ../../scripts/version-update.js",
+    "precompile": "cross-var lerna run version --scope $npm_package_name --include-dependencies",
+    "prewatch": "npm run precompile",
+    "peer-api-check": "node ../../scripts/peer-api-check.js",
+    "align-api-deps": "node ../../scripts/align-api-deps.js"
+  },
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14"
+  },
+  "publishConfig": {
+    "access": "restricted"
+  },
+  "devDependencies": {
+    "@opentelemetry/api": ">=1.0.0 <1.10.0",
```

Same here, does not implement the API so we can directly depend on the latest version.
```suggestion
    "@opentelemetry/api": "1.9.0",
```